src/client/pages/index/module.vue

```diff
@@ -64,39 +65,43 @@ if (hot) {
   </NavBar>
   <Container
     v-if="data && data.transforms"
-    class="grid grid-cols-[300px_3fr] overflow-hidden"
+    class="flex overflow-hidden"
   >
-    <div class="flex flex-col border-r border-main">
-      <div
-        class="border-b border-main px-3 py-2 text-center text-sm tracking-widest text-gray-400"
-      >
-        {{ inspectSSR ? 'SSR ' : '' }}TRANSFORM STACK
-      </div>
-      <template v-for="tr, idx of data.transforms" :key="tr.name">
-        <button
-          class="block border-b border-main px-3 py-2 text-left font-mono text-sm !outline-none"
-          :class="currentIndex === idx ? 'bg-main bg-opacity-10' : ''"
-          @click="index = idx.toString()"
+    <Splitpanes>
+      <Pane size="20" min-size="5" class="flex flex-col border-r border-main overflow-y-auto">
```

Comment:
Can we use `useLocalStorage` to memorize the size? You may reference to Nuxt DevTools approach